ext/re/re.xs

```diff
@@ -100,6 +101,8 @@ const struct regexp_engine wild_reg_engine = {
         my_re_op_compile,
 };
 
+#define _newSVbool(x) newSViv((x) ? 1 : 0)
```

Comment:
It turns out that a leading underscore in something visible to C is illegal, reserved for the implementation.  We need to change ours to trailing underscores

Reply:
Thanks, I never heard of that; fixed.